README.md

```diff
@@ -2,7 +2,9 @@
 
 The Enhanced GitHub Theme.
 
-Get it from [Add-ons for Firefox](https://addons.mozilla.org/en-US/firefox/addon/github-enhancing/)
+Get it from 
+- [Add-ons for Firefox](https://addons.mozilla.org/en-US/firefox/addon/github-enhancing/)
+- [Microsoft Edge-Add-ons](https://microsoftedge.microsoft.com/addons/detail/github-enhancing/ogkeblbkmfddfmdmaegnfcnpnjjijold)
```

Comment:
You can use the shorter link provided by Partner Center `https://microsoftedge.microsoft.com/addons/detail/ogkeblbkmfddfmdmaegnfcnpnjjijold`

Reply:
Done